### PR TITLE
[CC 110]Updating about this release and architecture docs

### PR DESCRIPTION
**Pull request by pubudu538**
## Purpose
Updated the following.

1. About this release
2. Architecture

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/docs/get-started/about-this-release.md [hunk 1]
```diff
@@ -100,6 +100,14 @@ The API Manager component addresses all aspects of API management in the platfor
 
      This feature allows you to generate data service definitions by pointing to an existing data source in the workspace without creating it from scratch.
 
+- **[Mock API implementation support considering API definition in Choreo Connect]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-rest-api-with-mock-impl/)**
+
+     This Mock API Implementation allows you to generate mock responses based on the examples provided in the OpenAPI specification that will mock the API responses. 
+
+- **[Git integration for Choreo Connect]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/git-integration/)**
+
+     This feature allows you to deploy APIs from a Git repository by starting the Choreo Connect pointing to source control systems such as GitHub, Gitlab, and Bitbucket.
```
Comment (samgnaniah) [inline en/docs/get-started/about-this-release.md:109]:
```suggestion
     This feature allows you to deploy APIs from a GitHub repository by starting Choreo Connect and pointing to source control systems such as GitHub, Gitlab, and Bitbucket.
```

File: en/docs/get-started/apim-architecture.md [hunk 2]
```diff
@@ -50,6 +50,14 @@ The API Gateway does the JWT token validation by validating the signature, issue
...
 
+#### Choreo Connect
+
+<a href="{{base_path}}/assets/img/deploy/mgw/choreo-connect-basic-architecture.png"><img src="{{base_path}}/assets/img/deploy/mgw/choreo-connect-basic-architecture.png" width="600" alt="auth0 token validation"></a>
+
+Choreo Connect (the API Microgateway) is a cloud-native, open-source, and developer-centric API gateway. It provides first-class support for Kubernetes while facilitating an array of API management quality of services (QoS), such as message security, rate-limiting, observability, and message mediation. Choreo Connect can be run in the cloud, on-premises, or in a hybrid environment, as long as it supports Kubernetes.
```
Comment (samgnaniah) [inline en/docs/get-started/apim-architecture.md:57]:
```suggestion
Choreo Connect (the API Microgateway) is a cloud-native, open-source, and developer-centric API Gateway. It provides first-class support for Kubernetes while facilitating an array of API management quality of services (QoS), such as message security, rate-limiting, observability, and message mediation. Choreo Connect can be run in the cloud, on-premises, or in a hybrid environment, as long as it supports Kubernetes.
```

File: en/docs/get-started/apim-architecture.md [hunk 3]
```diff
@@ -50,6 +50,14 @@ The API Gateway does the JWT token validation by validating the signature, issue
...
+
+Choreo Connect (the API Microgateway) is a cloud-native, open-source, and developer-centric API Gateway. It provides first-class support for Kubernetes while facilitating an array of API management quality of services (QoS), such as message security, rate-limiting, observability, and message mediation. Choreo Connect can be run in the cloud, on-premises, or in a hybrid environment, as long as it supports Kubernetes.
+
+Choreo Connect comprises three components: Router, Enforcer, and Adapter. The Router is the component in charge of directing traffic from various clients to the intended destination (service). Choreo Connect uses the [Envoy Proxy](https://www.envoyproxy.io/) as the core component for traffic routing. The Router is key to exposing APIs to external and remote clients. The policy enforcement engine, the Enforcer, performs token validation, rate limitation, analytics, and other validations. The Adapter serves as the control plane for both the Router and the Enforcer. The Choreo Connect can be used as a standalone gateway or with WSO2 API Manager as the control plane.
```
Comment (samgnaniah) [inline en/docs/get-started/apim-architecture.md:59]:
```suggestion
Choreo Connect comprises three components: Router, Enforcer, and Adapter. The Router is the component in charge of directing traffic from various clients to the intended destination (service). Choreo Connect uses the [Envoy Proxy](https://www.envoyproxy.io/) as the core component for traffic routing. The Router is key to exposing APIs to external and remote clients. The policy enforcement engine, which is the Enforcer, performs token validation, rate limitation, analytics, and other validations. The Adapter serves as the control plane for both the Router and the Enforcer. Choreo Connect can be used as a standalone Gateway or with WSO2 API Manager as the Control Plane.
```

**Comment by samgnaniah** [comment]
Will there be changes to the OPA description or are we using what we have there?

**Comment by pubudu538** [comment]
Let's use what we have there. No need to add those specifically.

**Review by samgnaniah — APPROVED** [review]

**Pull request merged**